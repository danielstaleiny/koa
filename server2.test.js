const mongoose = require("./mongoose");
const request = require("supertest");
const app = require("./server");

const User = require("./user");

// beforeAll(async () => {
//     console.log("Prepare objects");
// });
afterAll(async () => {
    await User.remove({}).exec();
    mongoose.mongoose.connection.close();
});

test("root route", async () => {
    const response2 = await request(app.callback())
        .post("/users")
        .send({
            text: "anon"
        });
    expect(response2.body.text).toMatchSnapshot();
    const response = await request(app.callback()).get("/users/5");
    expect(response.body).toMatchSnapshot();
});
