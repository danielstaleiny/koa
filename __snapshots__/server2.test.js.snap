// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root route 1`] = `"anon"`;

exports[`root route 2`] = `
Object {
  "message": "User not found",
}
`;
